### PR TITLE
On CI, api keys can be read from env variables

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -24,13 +24,18 @@ android {
         buildConfigField 'String', 'LIGHTNING_VERSION_NAME', '"4.2.3.1"'
         buildConfigField "int", "MINIMUM_WEBVIEW_VERSION", "55"
 
-        if (this.hasProperty("com.cliqz.lumen.vpnApiKey")) {
-            buildConfigField 'String', 'VPN_API_KEY', this.getProperty("com.cliqz.lumen.vpnApiKey")
+        def vpnApiKey = this.hasProperty("com.cliqz.lumen.vpnApiKey") ?
+                this.getProperty("com.cliqz.lumen.vpnApiKey") : System.getenv("vpnApiKey")
+        if (vpnApiKey != null) {
```
Comment (sharath-cliqz) [inline app/build.gradle]: `if (vpnApiKey) {`

File: app/build.gradle [hunk 2]
```diff
@@ -24,13 +24,18 @@ android {
...
+            buildConfigField 'String', 'VPN_API_KEY', '"' + vpnApiKey + '"'
```
Comment (sharath-cliqz) [inline app/build.gradle]: `buildConfigField 'String', 'VPN_API_KEY', '"$(vpnApiKey)"'`

File: app/build.gradle [hunk 3]
```diff
@@ -24,13 +24,18 @@ android {
...
         } else {
             buildConfigField 'String', 'VPN_API_KEY', '""'
         }
-        if (this.hasProperty("com.cliqz.lumen.revenuecatApiKey")) {
-            buildConfigField 'String', 'REVENUECAT_API_KEY', this.getProperty("com.cliqz.lumen.revenuecatApiKey")
+
+        def revenueCatApiKey = this.hasProperty("com.cliqz.lumen.revenueCatApiKey") ?
+                this.getProperty("com.cliqz.lumen.revenueCatApiKey") : System.getenv("revenueCatApiKey")
+        if (revenueCatApiKey != null) {
```
Comment (sharath-cliqz) [inline app/build.gradle]: `if (revenueCatApiKey) {`

File: app/build.gradle [hunk 4]
```diff
@@ -24,13 +24,18 @@ android {
...
+            buildConfigField 'String', 'REVENUECAT_API_KEY', '"' + revenueCatApiKey + '"'
```
Comment (sharath-cliqz) [inline app/build.gradle]: `buildConfigField 'String', 'REVENUECAT_API_KEY', '"$(revenueCatApiKey)"'`

File: app/build.gradle (hunk 2 above)
Comment (spacifici) [reply app/build.gradle]: @sharath-cliqz your suggestion will not work:
```groovy
buildConfigField 'String', 'VPN_API_KEY', "\"${vpnApiKey}\""
```

File: app/build.gradle (hunk 4 above)
Comment (spacifici) [reply app/build.gradle]:
```groovy
buildConfigField 'String', 'REVENUECAT_API_KEY', "\"${revenueCatApiKey}\""
```

you can not iterpolate strings literals starting with a single quote

File: app/build.gradle (hunk 2 above)
Comment (sharath-cliqz) [reply app/build.gradle]: Ahh. yes. Thank You @spacifici

**Comment by spacifici** [comment]
Please move this PR to master

**Pull request closed without merging**